src/plugins/telemetry/common/ebt_v3_endpoint/types.ts

```diff
@@ -7,9 +7,7 @@
  */
 
 export type {
-  Event,
-  EventType,
-  EventContext,
-  TelemetryCounter,
-  TelemetryCounterType,
-} from './types';
+  BuildShipperHeaders,
+  BuildShipperUrl,
+  BuildShipperUrlOptions,
+} from '@elastic/ebt/shippers/elastic_v3/server';
```

Comment:
Should this be imported from `common`?
```suggestion
} from '@elastic/ebt/shippers/elastic_v3/common';
```